photon-client/src/integTest/scala/com/linkedin/photon/ml/data/avro/ScoreProcessingUtilsTest.scala

```diff
@@ -31,22 +31,22 @@ class ScoreProcessingUtilsTest extends SparkTestUtils with TestTemplateWithTmpDi
       predictionScore = 1.0,
       label = Some(1.0),
       weight = Some(1.0),
-      idTypeToValueMap = Map(AvroFieldNames.UID -> "1", "id2" -> "2")),
+      idTypeToValueMap = Map(FieldNames.uid -> "1", "id2" -> "2")),
```

Comment:
Not sure what happened here, but `FieldNames` doesn't exist anymore - you got rid of it, so all of these should be `AvroFieldNames`